the-content-guide/readme.md

```diff
@@ -1,20 +1,108 @@
-> **Goal of this guide:** Helping you create the best content that you can make.
+> **Goal of this guide:** Helping you create the best content you can make.
 
 # How to create awesome content
 
 This guide is intended for external communication but can also be a source for our in-house content developers.
-The target audience are communication professionals. See it as a comprehensive guide, a white paper if you will, to show others our know-how. Because the product that we sell are blog posts or web articles, we should be able to display our skills.
+The target audience is communication professionals. See it as a comprehensive guide—a white paper if you will—to show others our know-how. Because the products that we sell are blog posts and web articles, we should be able to display our skills.
```

Comment:
'...we should be able to display our skills in creating content.' Rounds out the sentence.